### PR TITLE
Added menu button

### PR DESCRIPTION
**Pull request by REHERC**
This pull request contains code to create a menu button in the Arcade category (in the main menu)

It also removes the need for a `Dependencies` folder (using NuGet packages instead)

### CODE REVIEW

File: DistanceRando-Spectrum/DistanceRando.csproj [hunk 1]
```diff
@@ -39,7 +39,7 @@
     <Reference Include="Assembly-CSharp, Version=9.0.0.0, Culture=neutral, processorArchitecture=MSIL">
       <HintPath>..\packages\Centrifuge.GameSupport.Distance.3.0.7868.41513\lib\net35\Assembly-CSharp.dll</HintPath>
     </Reference>
-    <Reference Include="Centrifuge.Distance, Version=3.0.7868.41513, Culture=neutral, processorArchitecture=MSIL">
+    <Reference Include="Centrifuge.Distance, Version=3.0.7867.3370, Culture=neutral, processorArchitecture=MSIL">
```
Comment (Tiyenti) [inline DistanceRando-Spectrum/DistanceRando.csproj]: It looks like you may have made a mistake here? The assembly version for the GSL reference is mismatched and as a result, without removing and re-adding the reference, the project won't build for me.
```suggestion
    <Reference Include="Centrifuge.Distance, Version=3.0.7868.41513, Culture=neutral, processorArchitecture=MSIL">
```

File: DistanceRando-Spectrum/Metadata.cs [hunk 2]
```diff
@@ -7,7 +7,7 @@ internal static class Metadata
 		public const string RandomizerVersion = "A.2.1";
 		public const string LogicVersion = "1.0-alpha2";
 
-		public const string MenuButtonText = "Distandomizer";
+		public const string MenuButtonText = "Distrandomizer";
```
Comment (Tiyenti) [inline DistanceRando-Spectrum/Metadata.cs]: This works, although I think simplifying the button label would be cleaner.
```suggestion
		public const string MenuButtonText = "Randomizer";
```

**Review by Tiyenti — CHANGES_REQUESTED** [review]
Nice work with this, although couple things I noticed that should probably be changed before merging this in.

File: DistanceRando-Spectrum/Metadata.cs (hunk 2 above)
Comment (REHERC) [reply DistanceRando-Spectrum/Metadata.cs]: I wasn't sure about the text so feel free to change it :+1:

File: DistanceRando-Spectrum/DistanceRando.csproj (hunk 1 above)
Comment (REHERC) [reply DistanceRando-Spectrum/DistanceRando.csproj]: I don't know why it happened but this shouldn't be a problem

**Review by Tiyenti — APPROVED** [review]
Made the change I suggested and fixed the assembly reference, builds fine now. Will merge.

**Pull request merged**